Detections/MultipleDataSources/BrutforceAttemptOnAzurePortalAndAWSConsolAtSameTime.yaml

```diff
@@ -0,0 +1,106 @@
+id: 1f40ed57-f54b-462f-906a-ac3a89cc90d4
+name: Cross-Cloud Credential Brute Force Attempt

```

Comment:
What about also doing a Cross-Cloud password spray detection?
You could use similar logic to this but with a spray, or also look at instances where a spray is attempted across both AWS and Azure but is only sucessful in one of them.

Reply:
@petebryan,Title been changed as you recommended, 
logic changed for the spray attempted but expecting more noise as Azure has more number of usage then AWS.